src/EditorFeatures/VisualBasicTest/CodeActions/EncapsulateField/EncapsulateFieldTests.vb

```diff
@@ -582,5 +584,29 @@ End Enum
             Await TestMissingAsync(enumField)
 
         End Function
+
+        <WorkItem(7090, "https://github.com/dotnet/roslyn/issues/7090")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.EncapsulateField)>
+        Public Async Function ApplyCurrentMePrefixStyle() As Task
+            Await TestAsync("
+Class C
+    Dim [|i|] As Integer
+End Class
+", "
+Class C
+    Dim i As Integer
+
+    Public Property I1 As Integer
+        Get
+            Return Me.i
+        End Get
+        Set(value As Integer)
+            Me.i = value
+        End Set
+    End Property
+End Class
+",
+options:=New Dictionary(Of OptionKey, Object)() From {{New OptionKey(SimplificationOptions.QualifyFieldAccess, LanguageNames.VisualBasic), True}})
```

Comment:
I don't want to see "new Dictionary" anywhere in this CR except in a helper function :)